tests/utils.py

```diff
@@ -30,6 +30,9 @@
 # dice: any combination of valid dice, rolled or unrolled
 DICE_PATTERN = rf"( *((\d*d\d+(\w+[lh<>]?\d+)?( *{ROLLED_DICE_PATTERN})?)|\d+|( *[-+*/]))( *\[.*\])?)+( *= *`\d+`)?"
 
+# Matches a number without +/-
+NUMBER_PATTERN = rf"([0-9]+)"
```

Comment:
isn't this the same as `\d+`?

Reply:
It sure is. I'm still not great with late night regex.